gitlab/v4/objects/groups.py

```diff
@@ -184,11 +201,11 @@ def share(self, group_id, group_access, expires_at=None, **kwargs):
             "group_access": group_access,
             "expires_at": expires_at,
         }
-        self.manager.gitlab.http_post(path, post_data=data, **kwargs)
+        return self.manager.gitlab.http_post(path, post_data=data, **kwargs)
```

Comment:
Should we also document this new return in the docstring?

I'm thinking in the future we should just remove the docstring typing since it duplicates the hints but just wondering.

Reply:
Done!